docs/nodes/compute/advanced/local-testing.md

```diff
@@ -0,0 +1,99 @@
+# Running Instances on the Local Environment
+
+Aleph.im supports two popular and high-performance hypervisors for virtualization: Firecracker and Qemu. This guide outlines the steps to run instances on your local machine for development and testing purposes.
+
+## 0. Prerequisites
+
+- **SSH Keys:** Ensure you have a valid SSH key pair set up. You'll need the public key to enable SSH access to the instances.
+
+## 1. Configuration
+
+### **Configure Allocation Token and SSH Keys:**
+
+- Edit the `/etc/aleph/supervisor.env` file and add the following lines:
+
+  ``` bash
+  # Encrypted allocation token hash in sha1 format
```

Comment:
Is it really sha1 and not sha256 ?

Can you document how a user can generate it ?

Reply:
Yes, it's sha256, I already changed it. I don't think that show the users how to create a token will be the topic of this guide. I will include a demo hash with its decrypted value.